packages/core-app-api/api-report.md

```diff
@@ -394,7 +394,7 @@ export type GithubSession = {
   providerInfo: {
     accessToken: string;
     scopes: Set<string>;
-    expiresAt: Date;
+    expiresAt?: Date;
```

Comment:
This is a bugfix rather than a breaking change, as it has always been the actual runtime type